shared/templates/service_disabled/ansible.template

```diff
@@ -31,7 +31,7 @@
     enabled: "no"
     state: "stopped"
     masked: "yes"
-  when: '"{{{ DAEMONNAME }}}.socket" in socket_file_exists.stdout_lines[1]'
+  when: 'socket_file_exists.stdout_lines is search("{{{ DAEMONNAME }}}.socket")' 
```

Comment:
```suggestion
  when: 'socket_file_exists.stdout_lines is search("{{{ DAEMONNAME }}}.socket", multiline=True)' 
```
Here you are already using the `search` filter for searching for `{{{ DAEMONNAME }}}.socket` in the output of the previous command. It is good. You only need to make sure the search is considering multi-lines output.